File: Linguagem_C/Algoritmos/LogicaParaDesenvolvimentoDeProgramacaoDeComputadores/capitulo_3/exercicio3E.c
#include<stdio.h>

double calculaPrestacao(double valor, double taxa, int tempo){
  return valor+(valor*(taxa/100)*tempo);
}

int main(void){
  double valor,taxa;
  int tempo;

  printf("Digite o valor:");
  scanf("%lf",&valor);

  printf("Digite a taxa:");
  scanf("%lf",&taxa);

  printf("Digte o tempo:");
  scanf("%i",&tempo);

  printf("A taxa a ser cobrada eh: R$%.2lf\n",calculaPrestacao(valor,taxa,tempo));
  return 0;
}

File: Python/Algoritmos/LogicaParaDesenvolvimentoDeComputadores/Capitulo_3/exercicio3E.py
"""
calcular a prestacao de um bem em atraso utilizando a seguinte fomula PRESTACAO = VALOR + (VALOR*(TAXA/100)*TEMPO)
"""

valor = float(input("Digite o valor:"))
taxa = float(input("Digite a taxa:"))
tempo = float(input("Digite o tempo de atraso:"))

prestacao = valor + (valor*(taxa/100)*tempo)

print(f"A prestacao eh: R$ {prestacao}")
